scaling_performance/using_cluster_loader.adoc

```diff
@@ -0,0 +1,245 @@
+[[scaling-performance-using-cluster-loader]]
+= Using Cluster Loader
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[what-cluster-loader-does]]
+== What Cluster Loader Does
+
+Cluster Loader is a tool that deploys large numbers of various objects to a
+cluster, which creates user-defined cluster objects. Build, configure, and run
+Cluster Loader to measure performance metrics of your {product-title} deployment
+at various cluster states.
+
+[[installing-cluster-loader]]
+== Installing Cluster Loader
+
+Cluster Loader is included in the `atomic-openshift-tests` package. To install
+it, run:
+
+----
+$ yum install atomic-openshift-tests
+----
+
+After installation, the test executable `extended.test` is located in
```

Comment:
Since this is basically referring to the file name, maybe use file name markup here too?